src/movies.js

```diff
@@ -1,16 +1,132 @@
 // Iteration 1: All directors? - Get the array of all directors.
+function getAllDirectors(movies){
+       //code
+       let answer = movies.map((movie)=>{
```

Comment:
Try to be more specific when naming your variables. Name then something that will reflect the value stored. For this, something like directorsArr.